classes/class-wc-connect-taxjar-integration.php

```diff
@@ -887,6 +887,43 @@ public function append_base_address_to_customer_taxable_address( $address ) {
 		return array( $country, $state, $postcode, $city );
 	}
 
+	/**
+	 * This method is used to override the TaxJar result.
+	 *
+	 * @param object $taxjar_resp_tax TaxJar response object.
+	 * @param array  $body            Body of TaxJar request.
+	 *
+	 * @return object
+	 */
+	public function override_taxjar_tax( $taxjar_resp_tax, $body ) {
+		$new_tax_rate = apply_filters( 'woocommerce_services_override_tax_rate', $taxjar_resp_tax->rate, $taxjar_resp_tax, $body );
+
+		if ( $new_tax_rate !== $taxjar_resp_tax->rate ) {
```

Comment:
Let's revert condition check and do early return if `$new_tax_rate === $taxjar_resp_tax->rate` before comparing let's cast each value to float by using `floatval`